.pylintrc

```diff
@@ -293,7 +293,7 @@ ignored-classes=optparse.Values,thread._local,_thread._local,Dvcfile
 # (useful for modules/projects where namespaces are manipulated during runtime
 # and thus existing member attributes cannot be deduced by static analysis. It
 # supports qualified module names, as well as Unix pattern matching.
-ignored-modules=
+ignored-modules=azure
```

Comment:
@isidentical do we need it here?

Reply:
yes

Reply:
can we put a comment why do we need it here then? it's very hard to guess and will be very hard to understand when can we remove it

Reply:
the comment above seems to (partially) explain the situation, do you mean like a comment that points to the file this is required?

Reply:
yeah, may be it's me ... but for me it would be useful here to have something like - `azure - because we do .... in the .... and it's needed ....`

not a strong opinion, @isidentical .... just from my practice it's really hard to maintain all these settings, keep them up to date, etc.